test/integration/scipy_test.py

```diff
@@ -0,0 +1,136 @@
+# skip-if: True
+# This test will take a long time(51mins).
+# 3100.61s user 490.14s system 118% cpu 50:30.63 total
+# So please test it manually.
+
+import os
+import sys
+import subprocess
+
+"""
+Using this test file.
+
+SciPy C source code files generated individually by Cython, it may contains
+implicit declaration of functions when compiling. It will trigger errors like:
+"error: implicit declaration of function ‘zcopy_’ " when building SciPy.
+
+This option was disabled in CPython. But enabled in Pyston. So if you want build
+and test SciPy in Pyston, please revert the second change in
+https://github.com/dropbox/pyston/commit/949dfeb1f675ef05ddfd04333e7e6c0f9887860c
```

Comment:
My assumption has been that we would need to remove that flag eventually to bring ourselves in line with CPython.  But if there's some simple way to patch scipy to not need it I think that would be best for now :)